### ISSUE TITLE
Metadata block: title is not page title

### ISSUE THREAD

**Issue by kptdobe** [adobe/helix-pages#786]
The page title is at the moment the first heading of the page. I would expect that if you add the meta `title`, this would take over and override the "first heading compute" one. At the moment, it only drives the `og:title` which makes things inconsistent.

```md
# H1 Page title
...

| Metadata    |                       |
| ----------- | ------------- |
| Title       |  Meta Page Title| 
```

would lead to 

```html
<title>H1 Page Title</title>
<meta property="og:title" content="Meta Page Title">
...
<h1>H1 Page Title</h1>
```

I would expect the html to be:

```html
<title>Meta Page Title</title>
<meta property="og:title" content="Meta Page Title">
...
<h1>H1 Page Title</h1>
```

cc @rofe @davidnuescheler

**Comment by kptdobe** [comment]
I would still keep the h1 fallback in case no meta is provided.

**Comment by tripodsan** [comment]
should be fixed together with #800
